pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -54,8 +54,8 @@ var (
 	dashboardInitWaitTime = 5 * time.Second
 )
 
-func (c *Cluster) configureDashboardService(activeDaemon string) error {
-	dashboardService, err := c.makeDashboardService(AppName, activeDaemon)
+func (c *Cluster) configureDashboardService() error {
+	dashboardService, err := c.makeDashboardService(AppName)
```

Comment:
Testing this branch, I don't see the new `mgr_role=active` label on the dashboard service.

Reply:
> Testing this branch, I don't see the new `mgr_role=active` label on the dashboard service.

Please, can you give some more details about the test setup? (did you upgraded an existing version? which version was it? .. In the current code the label is inserted by the method `buildSelectorLabels` :

```
func (c *Cluster) buildSelectorLabels(labels map[string]string) map[string]string {
	selectorLabels := make(map[string]string)
	for k, v := range labels {
		selectorLabels[k] = v
	}
	selectorLabels["mgr_role"] = "active"
	return selectorLabels
}
```

which is called from: 

```
func (c *Cluster) makeDashboardService(name string) (*v1.Service, error) {

	labels := controller.AppLabels(AppName, c.clusterInfo.Namespace)
	selectorLabels := c.buildSelectorLabels(labels)
```
So I don't understand how the dashboard service doesn't have the `mgr_role` label.

Reply:
False alarm, I looked at the labels instead of the selectors.